support-frontend/package.json

```diff
@@ -90,6 +90,7 @@
     "@guardian/pasteup": "1.0.0-alpha.7",
     "@guardian/source": "1.0.1",
     "@guardian/source-development-kitchen": "1.0.0",
+    "@guardian/support-service-lambdas": "guardian/support-service-lambdas#3a4a8d7c841b713b736115772a9aadd05995b66a",
```

Comment:
what does the hash represent?

Reply:
[It's the commit hash of the repo](https://github.com/guardian/support-service-lambdas/commit/70e330b5c433184981e2c6955025cbaa7666f353).

Strangely I can't find the official documentation for the resolver, but it's documented here: https://www.warp.dev/terminus/npm-install-from-github#remote-package-by-branch

Reply:
I've updated it to be more explicit and use the whole URL to hopefully make this more clear.